File: queue/bank.c
#include <stdio.h>
#include "fila.h"

int main () {

    struct Fila f;
    int elemento;
    create(&f);

    inserir(&f, 176);
    inserir(&f, 914);
    inserir(&f, 1817);
    inserir(&f, 100);

    printf("bem vindo ao banco");
    printf("--------------------------------");
    
    while(!isEmpty(f)){
        elemento = retirar(&f);
        prinft("numero é:", elemento);
    }

    return 0;
}

File: queue/Bank.java
public class Banco{
    public static void main(String args[]){

        Fila f;

        f = new Fila();

        f.inserir(176);
        f.inserir(914);
        f.inserir(1817);
        f.inserir(100);
        
        System.out.println("Bem vindo ao banco");
        while(!f.isEmpty()){
            e = f.retirar();
            System.out.println("Senha"+e);
        }

    }
}